indico/modules/events/timetable/templates/move_timetable_entry.html

```diff
@@ -0,0 +1,86 @@
+<div class="action-box highlight">
+    <div class="section">
+        <div class="text">
+            {% if parent_entry %}
```

Comment:
What about actually rendering the text inside the `if` block? That would allow you to put it inside a `{% trans %}` block. You could repeat "Please select the place where you want to move it to" if needed. It's pretty confusing the way it is right now.